controllers/workloads/console/controller.go

```diff
@@ -676,7 +862,46 @@ func (r *ConsoleReconciler) buildJob(logger logr.Logger, name types.NamespacedNa
 	}
 }
 
-func buildRole(name types.NamespacedName, podName string) *rbacv1.Role {
+func buildServiceRole(name types.NamespacedName, podName string) *rbacv1.Role {
```

Comment:
Some comments about why we need to build and apply these roles/rolebindings wouldn't go amiss

Reply:
Maybe the comment could go on `createOrUpdateServiceRbac` though